### PR TITLE
Update merging and Zuul update docs

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

File: doc/ci_admin_guide/Update_Zuul.md [hunk 1]
```diff
@@ -28,7 +28,15 @@ This procedure updates configuration using `development` branch from [contrail-w
     cd contrail-windows-ci
     ```
 
-1.  Verify that `development` branch contains PRs with required changes to Zuul configuration
+1. **[ONLY IF UPDATING FROM `production` branch]** Checkout `production` branch
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: It's not clear what this means. Maybe something like:

```
1. Checkout branch that you want to use for configuring Zuul. For example, to checkout production branch: ...
```

File: doc/ci_admin_guide/Update_Zuul.md [hunk 2]
```diff
@@ -28,7 +28,15 @@ This procedure updates configuration using `development` branch from [contrail-w
...
+
+    ```bash
+    git fetch --all
+    git branch production origin/production
+    git checkout production
+    ```
+
+1.  Verify that `development` (or `production`) branch contains PRs with required changes to Zuul configuration
```
Comment (m-kostrzewa) [inline doc/ci_admin_guide/Update_Zuul.md]: If we give an example for production, let's stick to having production come up first (out of parentheses)

```
Verify that `production` (or `development`)...
```

File: doc/ci_admin_guide/Update_Zuul.md (hunk 1 above)
Comment (m-kostrzewa) [reply doc/ci_admin_guide/Update_Zuul.md]: Otherwise, it could mean that this step should only be performed for production branch.

**Review by m-kostrzewa — CHANGES_REQUESTED** [review]

**Review by jacek-izykowski — COMMENTED** [review]
Readability of `**[ONLY IF UPDATING FROM production branch]** Checkout production branch` could be improved as MKos suggests. However I expect everyone to understand what's this point about.

**Review by m-kostrzewa — APPROVED** [review]

**Pull request merged**